### ISSUE TITLE
Error when trying to start the bot

### ISSUE THREAD

**Issue by MauliQT** [AmericanPresidentJimmyCarter/yasd-discord-bot#69]
command in question:
cd src
python -m bot YOUR_DISCORD_BOT_TOKEN -g YOUR_GUILD_ID
trying to deploy in docker

![asdfasdf](https://user-images.githubusercontent.com/87561264/202534426-c3ff571a-cd06-4b1d-97af-690cfdb01d93.PNG)

**Comment by AmericanPresidentJimmyCarter** [comment]
You are running Python 2.7. You should be running Python 3.10+.

**Comment by MauliQT** [comment]
Still get an error with python3.10.8

<img width="736" alt="Screenshot 2022-11-18 at 07 55 37" src="https://user-images.githubusercontent.com/87561264/202640250-cf771a99-bfc5-44d1-a4cc-cf3c3f85c4d5.png">

**Comment by AmericanPresidentJimmyCarter** [comment]
did you `pip install requirements.txt`?

**Comment by MauliQT** [comment]
yes i did that still doesnt work it gives me the same error and says already satisfied

**Comment by AmericanPresidentJimmyCarter** [comment]
You should have a folder inside the `env` virtual environment folder called `discord` after `pip install`ing it, at `env/lib/python3.10/site-packages/discord`. Do you see this folder in the `yasd-discord-bot` directory?
